Assets/scripts/Tool.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class Tool : MonoBehaviour
+{
+    public ToolType toolType;
```

Comment:
Nu de ToolType public is:
 - Ga ik ervanuit dat je hem nog wilt wijzigen vanuit een externe class.

Ik neem aan dat dit niet de bedoeling is?

Reply:
yup